src/components/MarkdownTable.tsx

```diff
@@ -1,23 +1,21 @@
 import React from "react"
-import styled from "@emotion/styled"
-
-const TableContainer = styled.div`
-  overflow-x: auto;
-  margin: 2rem 0;
-
-  th {
-    border-bottom: 1px solid ${(props) => props.theme.colors.border};
-    white-space: nowrap;
-  }
-`
+import { Table, TableContainer, Th } from "@chakra-ui/react"
 
 export interface IProps {
   children?: React.ReactNode
 }
 
 const MarkdownTable: React.FC<IProps> = ({ children }) => (
-  <TableContainer>
-    <table>{children}</table>
+  <TableContainer my={8}>
+    <Table>
+      <Th
+        borderBottom={"1px solid"}
+        borderColor={"border"}
+        whiteSpace={"nowrap"}
```

Comment:
```suggestion
        borderBottom="1px solid"
        borderColor="border"
        whiteSpace="nowrap"
```

Reply:
Hey @pabloped thank you for this I resolved this issue. Turns out this issue happened when I used the `TableContainer` component so replaced it with `Box` and since the children is being rendered within the `Table` I decided to add the `sx{} `option to it to style the `Th` this also fixed the issue with the `borderBottom` being under the whole table